source/unified-test-format/unified-test-format.rst

```diff
@@ -1465,20 +1468,18 @@ The following arguments are supported:
 
 - ``session``: Required string. See `commonOptions_session`_.
 
-  The client entity associated with this session SHOULD specify true for
-  `useMultipleMongoses <entity_client_useMultipleMongoses_>`_. This is advised
-  because multiple mongos connections are necessary to test session pinning.
-
-The MongoClient and mongos on which to run the ``configureFailPoint`` command is
-determined by the ``session`` argument (after resolution to a session entity).
-Test runners MUST error if the session is not pinned to a mongos server at the
-time this operation is executed.
+Once the ``session`` argument has been resolved to a session entity, the test
+runner MUST create a new MongoClient that is directly connected to the
+session's pinned server. Test runners MUST error if the session is not pinned
+to a mongos server at the time this operation is executed. The new
+MongoClient MUST be used to run the ``configureFailPoint`` command and MUST
+be closed after the command has finished executing executing.
 
 When executing this operation, the test runner MUST keep a record of both the
-fail point and session (or pinned mongos server) so that the fail point can be
-disabled on the same mongos server after the test. The test runner MUST also
```

Comment:
The bit about ignoring the `configureFailPoint` command no longer applies because we're not creating a client entity that could be referenced in a `expectEvents` directive.